docs/user-guide/gpus.md

```diff
@@ -0,0 +1,136 @@
+---
+assignees:
+- vishh
+title: GPU Support
+---
+
+Kubernetes includes **experimental** support for managing NVIDIA GPUs spread across nodes.
+This page describes how users can consume GPUs and the current limitations.
+
+## Pre-requisites
+
+1. Kubernetes nodes have to be pre-installed with Nvidia drivers. Kubelet will not detect Nvidia GPUs otherwise.
```

Comment:
The drivers have to be installed for kubelet to detect GPUs. If kubelet does not include GPUs in capacity, then the drivers have to be (re)installed.